skins/oasis/css/core/PageHeader.scss

```diff
@@ -3,40 +3,44 @@
 @import 'skins/shared/mixins/flexbox';
 @import 'layout';
 
-.WikiaPageHeader {
-	margin: 0 10px 10px;
-	position: relative;
-
-	&.separator {
-		border-bottom: 1px solid $color-page-border;
-		padding: 0 0 15px;
+.page-header {
```

Comment:
Sorry, at the time I completely forgot font-size and line-height was set elsewhere. That, and other related styles, need to be updated to use the new class name.